orchestrator-salt/src/main/resources/salt-common/salt/fluent/settings.sls

```diff
@@ -36,6 +36,8 @@
 {% set s3_log_bucket = salt['pillar.get']('fluent:s3LogArchiveBucketName') %}
 {% set azure_container = salt['pillar.get']('fluent:azureContainer') %}
 {% set azure_storage_instance_msi = salt['pillar.get']('fluent:azureInstanceMsi') %}
+{% set azure_storage_instance_msi = salt['pillar.get']('fluent:azureIdBrokerInstanceMsi') %}
```

Comment:
Hmm, this is certainly wrong as it overwrites the value just retrieved in the previous line. 😄